price-estimator/src/models.rs

```diff
@@ -67,13 +67,45 @@ pub struct EstimatedBuyAmountResult {
     pub sell_amount_in_quote: u128,
 }
 
+#[derive(Debug, Serialize)]
+pub struct TransitiveOrder {
+    pub price: f64,
+    pub volume: f64,
+}
+
+impl From<&pricegraph::TransitiveOrder> for TransitiveOrder {
+    fn from(order: &pricegraph::TransitiveOrder) -> Self {
+        Self {
+            price: order.buy / order.sell,
+            volume: order.sell,
```

Comment:
I think this is correct based on https://github.com/gnosis/dex-price-estimator/blob/a7b1f5793e95501174865df140f783338213a107/src/orderbook_fetcher.ts#L109 but to be sure we'll need to wait until this is fully implemented and compare the output to the nodejs price estimator.

Reply:
This depends on which way the transitive order is expressed. We need the price in the quote token (2nd token in the pair) and the amount in base token (1st token in the pair). I think the way the [transitive orderbook](https://github.com/gnosis/dex-services/blob/6c707afc5810175514e3e152e075446cc6d47262/pricegraph/src/lib.rs#L15) is structured right now, it would have to differ between bids and asks.

I always find it confusing to now "what" exactly a price is expressing. @nlordell was mentioning that he was thinking about a refactoring to annotate prices with the pair (or at least the denominated token). This would have also helped me in https://github.com/gnosis/dex-services/pull/985#discussion_r439098477

Reply:
I see now, asks and bids need prices to be in the quote token, so for asks, its `order.buy / order.sell` and for bids its `order.sell / order.buy`.

Reply:
I have turned this into `unimplemented` for now and will add it back in the next PR.